gptcache/manager/vector_data/elastic_search.py

```diff
@@ -0,0 +1,87 @@
+from typing import List
+
+import numpy as np
+
+from gptcache.manager.vector_data.base import VectorBase, VectorData
+from gptcache.utils.log import gptcache_log
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import NotFoundError
+
+
+class ElasticSearchStore(VectorBase):
+    def __init__(
+        self,
+        host: str = "localhost",
+        port: str = "9200",
+        username: str = "",
+        password: str = "",
+        dimension: int = 0,
+        collection_name: str = "gptcache",
+        top_k: int = 1,
+        namespace: str = "",
+    ):
+        self._client = Elasticsearch("http://localhost:9200")
```

Comment:
@SimFG @cxie @xiaofan-luan 
I am currently running a local docker image of elastic search. What should be the hostname for the production use cases here?

Also, a point to note, I have used a `http` URL since, TLS was required to use a `https` with Elastic Search

Reply:
Perhaps through a variable, let the user pass the url parameter by himself. The default value of the parameter is: "http://localhost:9200"